test/SlowTests/Authentication/AuthenticationLetsEncryptTests.cs

```diff
@@ -209,14 +223,14 @@ private void SetupLocalServer()
             UseNewLocalServer(customConfigPath: settingPath);
         }
 
-        private async Task<X509Certificate2> GetCertificateFromLetsEncrypt(SetupInfo setupInfo, string acmeUrl)
+        private async Task<X509Certificate2> GetCertificateFromLetsEncrypt(TestingSetupInfo setupInfo, string acmeUrl)
```

Comment:
lets use proper naming so we don't endup with `setupInfo.setupInfo`
```suggestion
        private async Task<X509Certificate2> GetCertificateFromLetsEncrypt(TestingSetupInfo info, string acmeUrl)
```

Reply:
done